src/test/groovy/com/palantir/gradle/docker/PalantirDockerPluginTests.groovy

```diff
@@ -157,7 +157,7 @@ class PalantirDockerPluginTests extends AbstractPluginTest {
         buildResult.task(':dockerPrepare').outcome == TaskOutcome.SUCCESS
         buildResult.task(':docker').outcome == TaskOutcome.SUCCESS
         exec("docker inspect --format '{{.Author}}' ${id}") == "'${id}'\n"
-        exec("docker rmi ${id}")
+        exec("docker rmi -f ${id}")
```

Comment:
saw a consistent failure without the force flag when building this on circle; the `rmi` is just for cleanup, so happy to just punch the force button.